library/README.md

```diff
@@ -7,3 +7,5 @@ Here is list of updated modules:
 
 - Support target_network_dc attribute in vnic_profile_mappings: [PR](https://github.com/ansible/ansible/pull/41068)
   The fix will be available in Ansible 2.6.1.
```

Comment:
Please remove also lines above (6-9) as ovirt_vms module is no longer included within the role